nemo/collections/nlp/models/nlp_model.py

```diff
@@ -84,6 +84,8 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None, no_lm_init=False):
                     trainer=trainer,
                     cfg=copy_cfg,
                 )
+                # explicitly setting the is_model_being_restored to true
+                AppState().is_model_being_restored = True
```

Comment:
Please explain more in comments what is happening here and why we're doing line 88.

Reply:
At line no 81 while intializing the model , the states is_model_being_restore is set to false which is conflicting below when we again initailize the model at line 101 super().__init__(). Hence manually resetting the is_model_being_restore=True.
modified changelog as well